tests/test_travis_mf_multiplePS/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+for cmd in "timeout 60 ./tests/test_travis_mf/test_ps" "timeout 50 ./tests/test_travis_mf/worker" "timeout 50 ./tests/test_travis_mf/worker"; do
```

Comment:
Don't use this for loop (yes I know you copied this from previous tests). Just do each operation one by one. It's more readable that way

Reply:
Fixed.